docs/design/autoware-interfaces/ad-api/features/routing.md

```diff
@@ -35,3 +35,7 @@ See the sections below for supported options and details.
 ### allow_goal_modification
 
 **[v1.1.0]** Autoware tries to look for an alternate goal when goal is unreachable (e.g., when there is an obstacle on the given goal). When setting a route from the API, applications can choose whether they allow Autoware to adjust goal pose in such situation. When set false, Autoware may get stuck until the given goal becomes reachable.
+
+### allow_while_using_route
+
+**[v1.6.0]** This option only affects the route change APIs. Autoware accepts new route even while the vehicle is using the current route. The APIs fail if it cannot safely transition to new route. When set false, the APIs always fail when the vehicle is using the route.
```

Comment:
The current `/autoware_adapi_msgs` is v1.3.0. https://github.com/autowarefoundation/autoware_adapi_msgs/tags
So v1.4.0 seems to be better to bump up, doesn't it?

```suggestion
**[v1.4.0]** This option only affects the route change APIs. Autoware accepts new route even while the vehicle is using the current route. The APIs fail if it cannot safely transition to new route. When set false, the APIs always fail when the vehicle is using the route.
```

Reply:
@youtalk 
Your comment made me realize that there are two versions of the AD API: specification and messages. Since we are talking about the specification version here, v1.6.0 is correct. Because the specification version v1.5.0 has already been released.

I think it's necessary to create a correspondence table between specification and message versions, or to update the message version to match the specifications version even if there are no message changes. Which do you think is better?

Reply:
> update the message version to match the specifications version even if there are no message changes.

I think it is more clear. Please update package.xml to bump version to 1.6.0 even there is no actual updates.

Reply:
Okay, I'll always update the message version to match the spec version.

Reply:
https://github.com/autowarefoundation/autoware_adapi_msgs/pull/63